scripts/release/inline-resources.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+
+/**
+ * Simple Promiseify function that takes a Node API and return a version that supports promises.
+ */
+function promiseify(fn) {
+  return function() {
+    const args = [].slice.call(arguments, 0);
+    return new Promise((resolve, reject) => {
+      fn.apply(this, args.concat([function (err, value) {
+        if (err) {
+          reject(err);
+        } else {
+          resolve(value);
+        }
+      }]));
+    });
+  };
+}
+
+const readFile = promiseify(fs.readFile);
+const writeFile = promiseify(fs.writeFile);
+
+
+/**
+ * For every argument, inline the templates and styles under it and write the new file.
+ */
+for (let arg of process.argv.slice(2)) {
+  if (arg.indexOf('*') < 0) {
+    // Argument is a directory target, add glob patterns to include every files.
+    arg = path.join(arg, '**', '*');
+  }
+
+  const files = glob.sync(arg, {})
+    .filter(name => name.match(/\.js$/));  // Matches only JavaScript files.
```

Comment:
Should use either `RegExp.prorotype.test` or `String.prototype.search` rather than `match`